status-im/status-desktop
# chore: community join response with mvds

### What does the PR do

Make ApplicationMetadataMessage_COMMUNITY_REQUEST_TO_JOIN_RESPONSE message to use MVDS.

Status-go: https://github.com/status-im/status-go/pull/5787



### Affected areas



### Architecture compliance

My PR is consistent with this document: [Architecture guidelines](../architecture-guide.md)

### Screenshot of functionality (including design for comparison)






### How to test

The change should increase the stability of join community process.

- Create a community, test join community process without any error, with or without auto accept.
- Create Owner Token for the community, test join community process without any error, with or without auto accept.
- The change should be compatible with master

### Risk 

Described potential risks and worst case scenarios.

Tick **one**:
